geopmpy/test/TestPolicyStore.py

```diff
@@ -15,7 +15,7 @@
 class TestPolicyStore(unittest.TestCase):
     def setUp(self):
         mock_c.reset_mock()
-        with mock.patch('geopmpy.policy_store.gffi.get_dl_geopm', return_value=mock_c):
+        with mock.patch('geopmpy.gffi.dl_geopm', new=mock_c):
             reload(geopmpy.policy_store)
 
     def tearDown(self):
```

Comment:
This doesn't work, there is an import error if policy_store is not compiled into the library